prowler/lib/check/compliance.py

```diff
@@ -41,6 +43,64 @@ def update_checks_metadata_with_compliance(
                         check_compliance.append(compliance)
             # Save it into the check's metadata
             bulk_checks_metadata[check].Compliance = check_compliance
+
+        # Add requirements of Manual Controls
+        for framework in bulk_compliance_frameworks.values():
```

Comment:
why are you doing the same for again here? 

You can use the one above.

Reply:
Because the one above is iterating for each check and it has to be only done once.